packages/server/package.json

```diff
@@ -56,13 +56,13 @@
     "jsdom": "^15.1.1",
     "json-loader": "^0.5.7",
     "lint-staged": "^10.1.7",
-    "mocha": "^5.2.0",
+    "mocha": "^9.0.3",
```

Comment:
we don't use this anymore, so we could remove it entirely

Reply:
@mattkrick will take a card to setup a time to prune old/not-needed modules in the future. Leaving this for now.